src/data.js

```diff
@@ -2941,6 +2941,35 @@ module.exports = [
       'Motorcycle',
     ],
   },
+  {
+    name: 'Nikola Đuza',
+    description:
+      'Nikola helps developers improve their productivity by sharing pragmatic advice & applicable knowledge on JavaScript and Ruby.',
+    url: 'https://pragmaticpineapple.com/uses',
+    twitter: '@nikolalsvk',
+    emoji: '🍍',
+    country: '🇷🇸',
+    computer: 'apple',
+    phone: 'iphone',
+    tags: [
+      'Software Engineer',
+      'Developer',
+      'Writer',
+      'Front End',
+      'Back End',
+      'Full Stack',
+      'JavaScript',
+      'CSS',
+      'React',
+      'Ruby',
+      'Ruby on Rails',
+      'Developer',
+      'Speaker',
+      'Blogger',
+      'Teacher',
+      'Motorcycle',
```

Comment:
Looks great, just so you know Tags won't appear unless multiple people have them. "Motorcycle" might not show up.

Reply:
Thanks 🙇 I saw it from the person above me so I put it for fun 😂